### PR TITLE
Feature/mobile authentication

### PR DESCRIPTION
**Pull request by mh-izadi**

### CODE REVIEW

File: accounts/forms.py [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 
 
-class MyRegistrationForm(RegistrationForm):
-    invitee_invitation_code = forms.CharField(max_length=5, label="کد معرف")
+# class MyRegistrationForm(RegistrationForm):
+#     invitee_invitation_code = forms.CharField(max_length=5, label="کد معرف")
+#
+#     class Meta:
+#         model = Member
+#         fields = (UsernameField(), "email", "mobile", "address", "birth_date", "invitee_invitation_code")
```
Comment (LordST95) [inline accounts/forms.py]: remove commented code

File: accounts/templates/accounts/registration/register.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+{% block content %}
+
+    <div class="login-body">
+          <div class="container">
+
+              <form class="form-signin" action="" method="post">
+                  {% csrf_token %}
+                <h2 class="form-signin-heading">دریافت کد</h2>
+                <div class="login-wrap">
+{#                    <p>{{ error }}</p>#}
```
Comment (LordST95) [inline accounts/templates/accounts/registration/register.html]: if there's no need of these lines, plz remove them

File: accounts/templates/edit-profile.html [hunk 3]
```diff
@@ -65,13 +65,13 @@ <h4>
...
                                 </div>
                             </div>
-                            <div class="form-group">
-                                <label class="col-sm-2 control-label">شماره موبایل</label>
-                                <div class="col-sm-10">
-                                    <input id="id_customer_username" type="text" class="form-control" autofocus name="mobile" value="{{ object.mobile }}">
-                                    <span class="error-block">{{ form.mobile.errors }}</span>
-                                </div>
-                            </div>
+{#                            <div class="form-group">#}
```
Comment (LordST95) [inline accounts/templates/edit-profile.html]: again

**Comment by LordST95** [comment]
plz remove commented code and push one more time.

**Comment by mh-izadi** [comment]
done

**Comment by LordST95** [comment]
I appreciate your cooperation

**Pull request merged**